frontend/src/locales/da/alerts.json

```
@@ -29,6 +29,7 @@
     "V_MISSING_REQUIRED_VALUE": "Manglende værdi er påkrævet",
     "V_MISSING_START_DATE": "Manglende start dato",
     "V_MORE_THAN_ONE_ASSOCIATION": "Medarbejderen har allerede en aktiv tilknytning med organisationsenheden",
+    "V_MORE_THAN_ONE_PRIMARY": "Medarbejderen er allerede blevet tildelt primær",
```

Sentence ends abruptly?

```suggestion
    "V_MORE_THAN_ONE_PRIMARY": "Medarbejderen har allerede en primær funktion af denne slags",
```